src/shared/constants.ts

```diff
@@ -59,6 +60,7 @@ interface StacksChainConfig extends BaseChainConfig {
 export interface NetworkConfiguration {
   name: string;
   id: DefaultNetworkConfigurations;
+  testId?: string;
```

Comment:
I'm not sure I follow what's being solved by having the testId here?

Reply:
How is it being used for enabling test net? I was only able to find it being used for a `data-testid` attribute in `src/app/pages/select-network/components/network-list-item.layout.tsx`, did I miss other uses?

Reply:
https://github.com/hirosystems/wallet/pull/3410/files#diff-1bbef70f4a161b926410ea0225bff65a897632f6f261ff95e0734f87c2f81610R51

Reply:
Alright. I must be missing something, I don't understand the connection between the NetworkConfig object and the tests. Does the NetworkConfig object need to be aware of the app's testing needs? Would it make sense to decouple the network configuration from the test we're trying to write here?

Reply:
we get networks list from config, in order to display the networks. I'm not sure there is a suitable way to remove testIds from it 🤔 how we should define testIds in list items then? probably I could use network ids as testIds? is it ok to use them as selectors?

Reply:
Yes, I would recommend that rather than adding a new prop here. Use the `id` on `NetworkConfiguration`.

Reply:
done